### DISCUSSION TITLE
Allow passing only qburl in parameters

### DISCUSSION THREAD

**Discussion by StereotypicalCat** [jesec/flood#100] [Ideas]
Type: Feature Request

- [x] If you want to contribute to the project please review the [contributing guidelines](https://github.com/Flood-UI/flood/blob/master/.github/CONTRIBUTING.md).

## Summary

qBittorrent has an option to allow urls from localhost without authentication. However when auth = none, flood requires both url and username + password to launch.

## Idea of implementation

If only url is supplied, try to connect to torrent client without user and pass, and if it doesn't work, throw an exception saying the program needs a user and password. 
Alternatively, adding another parameter, along the lines of backend-auth = none

**Comment by jesec** [comment]
Does qBittorrent fail if (wrong) username or password was provided when its auth is disabled? If it doesn't, it might be simpler to just use random username/password.
